Sources/Public/DynamicProperties/AnyValueProvider.swift

```diff
@@ -42,51 +42,84 @@ extension AnyValueProvider {
 
 /// A base protocol for strongly-typed Value Providers
 protocol ValueProvider: AnyValueProvider {
-  associatedtype Value
+  associatedtype Value where Value: Interpolatable
 
   /// The strongly-typed storage for this Value Provider
   var storage: ValueProviderStorage<Value> { get }
 }
 
 extension ValueProvider {
-  public var typeErasedStorage: ValueProviderStorage<Any> {
+  public var typeErasedStorage: AnyValueProviderStorage {
     switch storage {
     case .closure(let typedClosure):
       return .closure(typedClosure)
+
     case .singleValue(let typedValue):
       return .singleValue(typedValue)
+
+    case .keyframes(let keyframes):
+      return .keyframes(keyframes.map { $0 }, interpolate: storage.value(frame:))
```

Comment:
What is the purpose of the identity mapping here? If there is one can we add a comment?
```suggestion
      return .keyframes(keyframes, interpolate: storage.value(frame:))
```

Reply:
This was because `KeyframeGroup<T>` isn't implicitly convertible to `KeyframeGroup<Any>`. (I removed the `KeyframeGroup.map` helper when I changed it back to being internal, so this isn't here anymore)